client/analysis.html

```diff
@@ -0,0 +1,26 @@
+
+
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="STEP Capstone">
+        <script src="js/analysis.js"></script>
```

Comment:
Consider moving the `script` tag to the end of your `body`. This way, your tag will have access to the DOM when it loads, instead of having to listen for a "page loaded" event.

Reply:
will do.

Reply:
Looks like this still needs to be resolved?